config/database.js

```diff
@@ -0,0 +1,11 @@
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize({
+  username: 'root',
+  password: null,
+  database: 'practice',
+  host: 'localhost',
+  dialect: 'mysql',
+});
+
```

Comment:
credentials/secrets are preferred to be saved in `.env` so that it won't be public. Please create a `.env` file and store credentials in it